### DISCUSSION TITLE
Read-only account

### DISCUSSION THREAD

**Discussion by azrossi** [firefly-iii/firefly-iii#5967] [General]
It'd be very useful to associate one or more read-only accounts to an existing account.
(of course granular permissions would be even more useful, but I guess a read-only account is faster to implement)
Use cases:
No-profit association which wants to share financials with its members or community
Condo admin who wants to share financials with home owners
etc...

**Comment by JC5** [comment]
Yes, this is on the roadmap for the new release, thanks for the suggestion!
